docs/source/getting_started_guide/introduction.rst

```diff
@@ -0,0 +1,16 @@
+++++++++++++++++++++++++
+Introduction to receptor
+++++++++++++++++++++++++
+
+Receptor is an overlay network.
+It eases the work distribution across a large and dispersed collection
+of workers
+
+Receptor nodes establish peer-to-peer connections with each other via
```

Comment:
```suggestion
Receptor nodes establish peer-to-peer connections with each other through
```

Reply:
Should avoid any Latin terms as it makes it more difficult for nonNative English speakers to understand. See https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#avoid-using-latin-phrases